### PR TITLE
#1254 Explain SLI overwrite logic

### PR DESCRIPTION
**Pull request by johannes-b**

### CODE REVIEW

File: content/docs/0.6.0/reference/monitoring/dynatrace/index.md [hunk 1]
```diff
@@ -161,22 +161,31 @@ During the evaluation of a quality gate, the Dynatrace SLI provider is required
...
+
+* In the below example, the SLI configuration as specified in the `sli-config-dynatrace.yaml` file is added to the service `carts` in stage `hardening` from project `sockshop`. 
+
   ```console
-  dynatrace-sli-service
-  dynatrace-sli-service-monitoring-configure-distributor
+  keptn add-resource --project=sockshop --stage=hardening --service=carts --resource=sli-config-dynatrace.yaml --resourceUri=dynatrace/sli.yaml
   ```
 
+**Note:** The add-resource command can be used to store a configuration on project-, stage-, or service-level. The overwriting strategy of Keptn is from the most generic to the more specific entity, meaning that a configuration on service-level overrides a configuration on stage-level, which overrides a configuration on the project-level. If, for example, an SLI is configured on service-level and on project-level, the configuration on service level overrules the other.
```
Comment (agrimmer) [inline content/docs/0.6.0/reference/monitoring/dynatrace/index.md]: Streamline: "service level" should be "service-level"

Comment (agrimmer) [reply content/docs/0.6.0/reference/monitoring/dynatrace/index.md]: I think the term "overwriting strategy of Keptn" is (too) complicated.
Can we replace 
"The overwriting strategy of Keptn..." 
to
"Keptn always first uses SLIs contained in the service-level, then Keptn uses SLIs contained in the stage-level, and finally Keptn uses SLIs contained in the project-level."

File: content/docs/0.6.0/reference/monitoring/prometheus/index.md [hunk 2]
```diff
@@ -77,22 +77,16 @@ During the evaluation of a quality gate, the Prometheus SLI provider is required
   kubectl get pods -n keptn | grep prometheus-sli
```
Comment (agrimmer) [inline content/docs/0.6.0/reference/monitoring/prometheus/index.md]: Here also please use a selector

File: content/docs/0.6.0/reference/monitoring/prometheus/index.md [hunk 3]
```diff
@@ -77,22 +77,16 @@ During the evaluation of a quality gate, the Prometheus SLI provider is required
...
+
+To tell the *prometheus-sli-service* how to acquire the values of an SLI, the correct query needs to be configured. This is done by adding an SLI configuration to a project, stage, or service using the [add-resource](../../cli/#keptn-add-resource) command. The resource identifier must be `prometheus/sli.yaml`.
+
+* In the below example, the SLI configuration as specified in the `sli-config-prometheus.yaml` file is added to the service `carts` in stage `hardening` from project `sockshop`. 
+
+  ```console
+  keptn add-resource --project=sockshop --stage=hardening --service=carts --resource=sli-config-prometheus.yaml --resourceUri=prometheus/sli.yaml
+  ```
+
+**Note:** The add-resource command can be used to store a configuration on project-, stage-, or service-level. The overwriting strategy of Keptn is from the most generic to the more specific entity, meaning that a configuration on service-level overrides a configuration on stage-level, which overrides a configuration on the project-level. If, for example, an SLI is configured on service-level and on project-level, the configuration on service level overrules the other.
```
Comment (agrimmer) [inline content/docs/0.6.0/reference/monitoring/prometheus/index.md]: See above.

File: content/docs/0.6.0/reference/monitoring/dynatrace/index.md [hunk 4]
```diff
@@ -161,22 +161,31 @@ During the evaluation of a quality gate, the Dynatrace SLI provider is required
   kubectl apply -f service.yaml
   ```
 
-**Verify deployment in your cluster:**
-
- Execute the following commands to verify the deployment of the Dynatrace SLI provider:
+* To verify that the deployment has worked, execute:
 
   ```console
   kubectl get pods -n keptn | grep dynatrace-sli
```
[the remark this answers is not in the document]
Comment (johannes-b) [reply content/docs/0.6.0/reference/monitoring/dynatrace/index.md]: Great idea! I changed it.

File: content/docs/0.6.0/reference/monitoring/dynatrace/index.md (hunk 1 above)
Comment (johannes-b) [reply content/docs/0.6.0/reference/monitoring/dynatrace/index.md]: I changed the wording and removed complexity.

File: content/docs/0.6.0/reference/monitoring/prometheus/index.md (hunk 2 above)
Comment (johannes-b) [reply content/docs/0.6.0/reference/monitoring/prometheus/index.md]: Changed it.

File: content/docs/0.6.0/reference/monitoring/prometheus/index.md (hunk 3 above)
Comment (johannes-b) [reply content/docs/0.6.0/reference/monitoring/prometheus/index.md]: Changed it.

**Pull request merged**